ci-operator/step-registry/monitoring-plugin/tests/perses-ui-ivt/monitoring-plugin-tests-perses-ui-ivt-commands.sh

```
@@ -0,0 +1,102 @@
+# Set Kubeconfig var for Cypress.
+cp -L "$KUBECONFIG" /tmp/kubeconfig && export CYPRESS_KUBECONFIG_PATH=/tmp/kubeconfig
+
+# Set Cypress base URL var.
+console_route=$(oc get route console -n openshift-console -o jsonpath='{.spec.host}')
+export CYPRESS_BASE_URL=https://$console_route
+
+# Set Cypress authentication username and password.
+# Use the IDP once issue https://issues.redhat.com/browse/OCPBUGS-59366 is fixed.
+export CYPRESS_LOGIN_IDP_DEV_USER=uiauto-htpasswd-idp
+export CYPRESS_LOGIN_IDP=kube:admin
+export CYPRESS_LOGIN_USERS=kubeadmin:${kubeadmin_password},${users}
```

Shellcheck (0.11.0)

[warning] users is referenced but not assigned.

(SC2154)